app/src/main/res/layout/view_fade_omnibar.xml

```diff
@@ -373,7 +373,7 @@
             android:layout_height="wrap_content"
             android:orientation="horizontal"
             android:gravity="center_horizontal"
-            android:visibility="gone"
```

Comment:
does this need to be added to the bottom one?

Reply:
Addressed in https://github.com/duckduckgo/Android/pull/5782.